packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -187,6 +187,18 @@ export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
      * Dispose of the container instance, permanently disabling it.
      */
     dispose(): void;
+
+    /**
+     * Gets the underlying {@link @fluidframework/container-definitions#IContainer}.
+     *
+     * FOR INTERNAL USE ONLY.
+     *
+     * @remarks Used to power debug tooling.
+     * @privateRemarks TODO: make this required in the future (breaking change).
+     *
+     * @internal
+     */
+    getInternalContainer?: () => IContainer;
```

Comment:
@skylerjokiel Interested in your thoughts on naming here. I went with "InternalContainer" initially, but perhaps "InnerContainer" is better? Or something else?